src/main/java/Event.java

```diff
@@ -0,0 +1,10 @@
+public class Event extends Task{
+    protected String time;
```

Comment:
It is better to make it a private string to ensure data hiding in encapsulation.